network-protection/network-protection-store/src/main/java/com/duckduckgo/networkprotection/store/waitlist/NetPWaitlistRepository.kt

```diff
@@ -14,19 +14,14 @@
  * limitations under the License.
  */
 
-package com.duckduckgo.networkprotection.impl.waitlist.store
+package com.duckduckgo.networkprotection.store.waitlist
```

Comment:
Mainly because I need it to set the auth token in the `network-protection-subscription` module. I can not use the repository and go directly to the datastore but no strong opinions since this is temporary.

Reply:
If the new module is an internal module, it should depend on the `impl` module. As we do for other internal modules.
Otherwise we start moving things around that should otherwise not be moved.